canonical/ubuntu.com
# remove 'Your organisation' section from /server and /management forms - regression

## Done

removed the 'Your org' section from forms and deleted the include
## QA
1. go to /server/contact-us or /management/contact-us
2. see there is no 'Your organisation' section
## Issue / Card

Fixes #425
